test/lib/absinthe/phase/document/flatten_test.exs

```diff
@@ -31,13 +31,9 @@ defmodule Absinthe.Phase.Document.FlattenTest do
       foo(id: $id) {
         gender
         ... FooFields
-        ... NotFooFields
         ... on Foo {
           age
         }
-        ... on NotFoo {
-          age
-        }
```

Comment:
These had to be removed because it isn't valid to query for not foo fields on foo. However we could add them back if we make `foo` more abstract, and the `notFoo` fields simply additions instead of NOT foo. I don't remember enough about what these are testing.